zipkin-common/src/test/scala/com/twitter/zipkin/storage/SpanStoreSpec.scala

```diff
@@ -120,8 +120,6 @@ abstract class SpanStoreSpec extends JUnitSuite with Matchers {
     result(store.getTraces(QueryRequest("service", annotations = Set("custom")))) should be(
       Seq(Seq(span1))
     )
-    // should not find any traces since the core annotation doesn't exist in index
```

Comment:
goofy and completely unnecessary. Many indexes cannot be configured to exclude certain words, so this is essentially busy-work to enforce.